docs/releases/v0.0.3.md

```diff
@@ -0,0 +1,88 @@
+# Auth Module Release v0.0.3
+
+Welcome to the latest release of the Auth module for the Kubernetes Fury Distribution.
+
+This is a maintenance release to update the Pomerium package. This update includes [breaking changes](#breaking-changes-) in Pomerium.
+
+## Included packages
+
+| Package    | Current Version                                                           | Previous Version |
+| ---------- | ------------------------------------------------------------------------- | ---------------- |
+| `dex`      | [`v2.35.3`](https://github.com/dexidp/dex/releases/tag/v2.35.3)           | No update        |
+| `gangway`  | [`v3.2.0`](https://github.com/vmware-archive/gangway/releases/tag/v3.2.0) | No update        |
+| `pomerium` | [`v0.21.0`](https://github.com/pomerium/pomerium/releases/tag/v0.21.0)    | `0.15.8`         |
```

Comment:
Should this be 0.19?

Reply:
No, I think `0.15.8` is ok because the previous release of the `auth` module contains that version of Pomerium.
But we should change the branch name before merging it, just for clarity.

Reply:
as @sbruzzese902 say, the version of pomerium included in auth v0.0.2 was 0.15.8.

I started working on this PR when the latest version of pomerium available was 0.19 but then I paused the work on the PR. When I started working again on this, the latest version was 0.21 and I updated everything but the branch name. Sorry for the confusion.

If you want I can rename the branch, but if we merge the PR we'd delete the branch anyway, so I don't see the need. Let me know what you prefer.